src/de/jost_net/JVerein/io/FormularAufbereitung.java

```diff
@@ -582,5 +632,120 @@ public void printAnschreiben(Spendenbescheinigung spb, String text)
       throw new RemoteException("Fehler", e);
     }
   }
-  
+
+  @SuppressWarnings("resource")
+  public void addZUGFeRD(Rechnung re, boolean mahnung) throws IOException
+  {
+    ArrayList<Mitgliedskonto> mklist = re.getMitgliedskontoList();
+    if (mklist.size() == 0)
+      return;
+
+    String sourcePDF = f.getAbsolutePath();
+    Einstellung e = Einstellungen.getEinstellung();
+    IZUGFeRDExporter ze = new ZUGFeRDExporterFromPDFA().ignorePDFAErrors()
+        .load(sourcePDF).setProducer("JVerein")
+        .setCreator(System.getProperty("user.name"));
+
+    Invoice invoice = new Invoice()
+        // F�lligkeitsdatum
+        .setDueDate(mklist.get(mklist.size() - 1).getDatum())
+        // Lieferdatum
+        .setDeliveryDate(mklist.get(mklist.size() - 1).getDatum())
+        // Rechnungsdatum
+        .setIssueDate(re.getDatum())
+        // Rechnungsnummer
+        .setNumber(re.getID());
+
+    // Rechnungssteller
+    TradeParty sender = new TradeParty(e.getName(),
+        StringTool.toNotNullString(e.getStrasse()),
+        StringTool.toNotNullString(e.getPlz()),
+        StringTool.toNotNullString(e.getOrt()), e.getStaat())
+            .addTaxID(e.getSteuernummer());
```

Comment:
Sollte hier noch der Adresszusatz gesetzt werden

Reply:
Ich sehe gerade, das fehlt in den Einstellungen Allgemein, dann kann man es auch nicht ausgeben.